lib/requestWithOAuth2.js

```diff
@@ -63,15 +63,19 @@ function doOAuth(oauthUrl, clientId, clientSecret, username, password, callback)
   }, callback);
 }
 
-function doRequestWithAccessToken(url, accessToken, callback) {
-  var authHeader = 'Bearer ' + accessToken;
-
-  request.get({
-    url: url,
-    headers: {
-      Authorization: authHeader
+function doRequestWithAccessToken(requestGetArgs, accessToken, callback) {
+  var authHeader = {Authorization: 'Bearer ' + accessToken};
```

Comment:
It's an object with only one property (representing the auth header). This distinction is analogous to a list element and a singleton list (a list with only one element).

Your code would still work if a future programmer added another header into the object which is why I think the name is now misleading.